src/pages/home/report/ReportActionsView.js

```diff
@@ -78,13 +79,15 @@ function ReportActionsView(props) {
     const prevNetworkRef = useRef(props.network);
     const prevIsSmallScreenWidthRef = useRef(props.isSmallScreenWidth);
 
+    const {isOffline} = useNetwork();
     const isFocused = useIsFocused();
     const reportID = props.report.reportID;
 
     /**
      * @returns {Boolean}
      */
     const isReportFullyVisible = useMemo(() => getIsReportFullyVisible(isFocused), [isFocused]);
+    const isFullHistoryFetched = useMemo(() => _.last(props.reportActions).actionName === CONST.REPORT.ACTIONS.TYPE.CREATED, [props.reportActions]);
```

Comment:
I don't think `_.last(props.reportActions).actionName === CONST.REPORT.ACTIONS.TYPE.CREATED` is expensive enough to be using `useMemo`. This is just accessing the lsat member of an array which should be very fast.